analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -125,7 +125,7 @@ private ProjectBuilder AddProject(AnalyzerLanguage language, string projectName,
             compilationOptions = languageName switch
             {
                 LanguageNames.CSharp => ((CSharpCompilationOptions)compilationOptions).WithAllowUnsafe(true),
-                LanguageNames.VisualBasic => ((VisualBasicCompilationOptions)compilationOptions).WithGlobalImports(GlobalImport.Parse(DefaultGlobalImportsVisualBasic)),
+                LanguageNames.VisualBasic => ((VisualBasicCompilationOptions)compilationOptions).WithGlobalImports(DefaultGlobalImportsVisualBasic),
```

Comment:
Related to https://github.com/SonarSource/sonar-dotnet/pull/4701#discussion_r672121192